testsuite/cluster-test/src/instance.rs

```diff
@@ -283,6 +284,9 @@ impl Instance {
             .arg("-c")
             .arg(command)
             .kill_on_drop(true)
+            .stdout(Stdio::null())
+            .stderr(Stdio::null())
+            .stdin(Stdio::null())
```

Comment:
@andll  Otherwise it pollutes output of the `cti` tool.

Reply:
For some use cases we actually want to see stdout/err.
I wonder if we can make add boolean argument(like tee_output) to exec function and set it to false in backup, but keep it in other use case?

Reply:
k. sure.